src/backend/chat_service/chat_service.py

```diff
@@ -118,15 +118,18 @@ async def upload_File(request: Request, auth=Depends(auth_admin_dependency), fil
             logger.error("Invalid or missing customer_guid in token")
             raise HTTPException(status_code=404, detail="Invalid customer_guid provided")
 
```

Comment:
Can we check whether the filename already exists in the system. 
If yes, throw an appropriate error